<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProxVDI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo/logo_32x32.png</normaloff>logo/logo_32x32.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="logo_label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 19, 47);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../logo/logo_breit.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_line"/>
      </item>
      <item>
       <widget class="QPushButton" name="reload_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/reload.svg</normaloff>../icons/reload.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="vm_listview">
      <property name="styleSheet">
       <string notr="true">QListView {
            border: 2px solid #8f8f91;  /* Rahmenfarbe */
            border-radius: 10px;  /* Abrundung der Ecken */
            background-color: #f0f0f0;  /* Hintergrundfarbe */
        }
        QListView::item {
            padding: 5px;  /* Abstand innerhalb der Listenelemente */
        }
        QListView::item:selected {
            background-color: #6699ff;  /* Hintergrundfarbe f체r ausgew채hlte Elemente */
            color: white;  /* Schriftfarbe f체r ausgew채hlte Elemente */
        }</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionSetting"/>
    <addaction name="actionLogout"/>
    <addaction name="actionLogin"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>vm_listview</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
